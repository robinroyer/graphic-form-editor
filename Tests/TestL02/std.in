R r01 10 10 5 5
LOAD save.txt
LOAD save02.txt
LIST
EXIT