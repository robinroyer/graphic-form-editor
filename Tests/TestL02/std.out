OK
#New object: r01
R r01 10 10 5 5
